src/resources/texture.js

```diff
@@ -82,12 +82,32 @@ pc.extend(pc, function () {
 
                 image.src = url;
             } else {
-                // Unsupported texture extension
-                // Use timeout because asset events can be hooked up after load gets called in some
-                // cases. For example, material loads a texture on 'add' event.
-                setTimeout(function () {
-                    callback(pc.string.format("Error loading Texture: format not supported: '{0}'", ext));
-                }, 0);
+                var blobStart = urlWithoutParams.indexOf("blob:");
+                if (blobStart >= 0) {
+                    urlWithoutParams = urlWithoutParams.substr(blobStart, urlWithoutParams.length - blobStart);
```

Comment:
@guycalledfrank 

Extract characters to the end of the string, so you can omit 2nd argument.
```js
urlWithoutParams = urlWithoutParams.substr(blobStart);
```